src/web/server/app/controllers/apiGroups.ts

```diff
@@ -0,0 +1,153 @@
+import { Request, Response } from 'express';
+
+import * as authServices from '../services/auth';
+import * as apiGroupsServices from '../services/apiGroups';
+import * as apiServices from '../services/api';
+
+/**
+ * Adds a user to a group
+ */
+export const addUserGroup = async (req: Request, res: Response) => {
+  try {
+    const userResult = await authServices.retrieveUser(req.body.username);
```

Comment:
Should check if `req.body.username` exists first

Reply:
yes